src/realm/sync/noinst/client_impl_base.hpp

```diff
@@ -1550,17 +1550,16 @@ inline void ClientImpl::Session::initiate_rebind()
 inline void ClientImpl::Session::reset_protocol_state() noexcept
 {
     // clang-format off
-    m_enlisted_to_send                    = false;
-    m_bind_message_sent                   = false;
-    m_error_to_send                       = false;
-    m_ident_message_sent = false;
-    m_unbind_message_sent = false;
-    m_unbind_message_send_complete = false;
-    m_error_message_received = false;
-    m_unbound_message_received = false;
-    m_client_error = util::none;
-
-    m_upload_progress = m_progress.upload;
+    m_enlisted_to_send                 = false;
```

Comment:
why did this all change?

Reply:
Reverted - I had reformatted the function when I was playing around with moving the bootstrap apply to this funciton.